src/main/resources/org/tosca/docs/messages.properties

```diff
@@ -22,3 +22,6 @@ feature.default=Default
 feature.description=Description
 feature.constraints=Constraints
 feature.derivedFrom=Derived from
+headers.interfaces=Interfaces
```

Comment:
I think these should also be added to the custom messages file.
The custom messages is used in a test that verifies that all the string labels are replaceable. 
Please also include the interfaces section in the i18n test